app/containers/message/Attachments.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import isEqual from 'lodash/isEqual';
+import PropTypes from 'prop-types';
+
+import Image from './Image';
+import Audio from './Audio';
+import Video from './Video';
+import Reply from './Reply';
+
+const Attachments = React.memo(({
+	attachments, timeFormat, user, baseUrl, useMarkdown, onOpenFileModal, getCustomEmoji
+}) => {
+	if (!attachments || attachments.length === 0) {
+		return null;
+	}
+
+	return attachments.map((file, index) => {
+		if (file.image_url) {
+			return <Image key={file.image_url} file={file} user={user} baseUrl={baseUrl} onOpenFileModal={onOpenFileModal} getCustomEmoji={getCustomEmoji} useMarkdown={useMarkdown} />;
```

Comment:
Nope.
I'll refactor soon with React.context in order to stop passing these props down the tree.
I didn't do it yet because I need `useMemo`.
Read this: https://kentcdodds.com/blog/how-to-optimize-your-context-value

Also: make sure to read all of Kent's posts. He's an awesome hooman :)